cli/azd/cmd/auth_login.go

```diff
@@ -290,8 +290,9 @@ func (la *loginAction) Run(ctx context.Context) (*actions.ActionResult, error) {
 			default:
 				panic("Unhandled login status")
 			}
-
-			fmt.Fprintln(la.console.Handles().Stdout, msg)
+			// get logged in user account information
+			act, _ := la.authManager.GetSignedInAccount(ctx)
```

Comment:
I think that if you are logged in with a service principal, I think that `GetSignedInAccount` will return `nil, nil` since we don't have a HomeAccountId for the current user.

We should do two things here:

1. Not ignore the returned error, but instead just log it and not display this message at all when this returns an error.
2. Handle the case where act is `nil`, I suspect by also not printing this?  I could imagine that printing the client ID of the service principal might be useful in that case.  We'd likely need to add some new method to the AuthManager to get this information in that case.

Reply:
on it.